src/components/UserTabs/ListItems/ListItems.jsx

```diff
@@ -1,5 +1,13 @@
-const ListItems = () => {
-  return <div>ListItems</div>;
+import UserCard from '../UserCard/UserCard';
+
+const ListItems = ({ users, tabType }) => {  
+  return (
+    <ul>
+      {users.map(({id, name, avatar_preview, recipes}) => (
+        <li key={id}><UserCard name={name} avatar_preview={avatar_preview} recipes={recipes} tabType = {tabType} /></li>
```

Comment:
li should be inside UserCard
pass the whole user to UserCard

Reply:
corrected